test/perf/merge.js

```diff
@@ -44,35 +44,35 @@ var options = {
 
 suite
   .add('most', function(deferred) {
-    var streams = a.map(most.from);
-    runners.runMost(deferred, most.mergeArray(streams).reduce(sum, 0));
+    const streams = map(from, a);
```

Comment:
I had to stare at it for a bit!  Pretty hard to see it after you've gotten accustomed to using function APIs everywhere :)